xbmc/cores/VideoPlayer/AudioSinkAE.h

```diff
@@ -74,7 +74,7 @@ class CAudioSinkAE : IAEClockCallback
   double m_timeOfPts;
   double m_syncError;
   unsigned int m_syncErrorTime;
-  double m_resampleRatio;
+  double m_resampleRatio = 0.0;
```

Comment:
I like the idea of invalid, this would trigger an error if not set later. mind a small comment?